recipes/datastax.rb

```diff
@@ -35,8 +35,8 @@
   action :install
 end
 
-package node['cassandra']['dsc_version'] do
-  action :install
+execute "install_cassandra" do
+  command "apt-get install -y #{node[:cassandra][:dsc_version]}=#{node[:cassandra][:version]}* cassandra=#{node[:cassandra][:version]}"
```

Comment:
Have you tested this fix works? We will want to test that before merging for the sake of community users of the cookbook.